lib/circles/create.js

```diff
@@ -3,10 +3,10 @@ const fullStateEnum = require("./fullStateEnum");
 module.exports = function(req, res) {
     const circle = {
         name: req.body.name || null,
-        vision: req.body.vision || null,
-        mission: req.body.mission || null,
-        aim: req.body.aim || null,
-        expectationsForMembers: [],
+        vision: req.body.vision || "",
+        mission: req.body.mission || "",
+        aim: req.body.aim || "",
+        expectationsForMembers: req.body.expectationsForMembers || "",
```

Comment:
In the current version they are there, though. But stored explicitly as null. (As evidenced by them being returned as null from GET /circle/{id}.)